tools/profile_endpoints.py

```diff
@@ -274,7 +278,44 @@ def profile_get_concepts(token):
 
 # Main function
 if __name__ == "__main__":
-    authenticate()
+
+    # Create the parser
+    parser = argparse.ArgumentParser(
+        description="Profile endpoints for a specific host and dataset"
+    )
+
+    # Add the arguments
+    parser.add_argument(
+        "--host",
+        type=str,
+        choices=HOSTS.keys(),
+        default="local",
+        help="The host to profile",
+    )
+
+    parser.add_argument(
+        "--dataset",
+        type=str,
+        choices=DATASETS.keys(),
```

Comment:
For local, I would expect we might want to offer more IDs? Is there a suggested approach for that?

Reply:
I would say, whoever needs that can add it then, or they could edit it locally and hardcode a different value.

I think most users will want to profile "large" data which means that they'd probably use ones in dev or (as in my case) ones in dev that I copied down to my local database.

Reply:
In that case should we not have local as an option for host? Or maybe keep local as an option but have it error if you try to call local with a dev dataset. Either way, this seems totally fine for now.

Reply:
> keep local as an option but have it error if you try to call local with a dev dataset

Actually that was my use case -- using `local` with the datasets in this file that I initialized locally using the dummy-dataset script here: https://drive.google.com/file/d/1RHBvVHLlXRl0o-g8ITN9_fjBMkeiJCgN/view